internal/storage/v2/clickhouse/tracestore/testdata/spans.go

```diff
@@ -10,34 +10,44 @@ import (
 )
 
 type SpanRow struct {
-	ID                     string
-	TraceID                string
-	TraceState             string
-	ParentSpanID           string
-	Name                   string
-	Kind                   string
-	StartTime              time.Time
-	StatusCode             string
-	StatusMessage          string
-	RawDuration            int64
-	BoolAttributeKeys      []string
-	BoolAttributeValues    []bool
-	DoubleAttributeKeys    []string
-	DoubleAttributeValues  []float64
-	IntAttributeKeys       []string
-	IntAttributeValues     []int64
-	StrAttributeKeys       []string
-	StrAttributeValues     []string
-	ComplexAttributeKeys   []string
-	ComplexAttributeValues []string
-	EventNames             []string
-	EventTimestamps        []time.Time
-	LinkTraceIDs           []string
-	LinkSpanIDs            []string
-	LinkTraceStates        []string
-	ServiceName            string
-	ScopeName              string
-	ScopeVersion           string
+	ID                          string
+	TraceID                     string
+	TraceState                  string
+	ParentSpanID                string
+	Name                        string
+	Kind                        string
+	StartTime                   time.Time
+	StatusCode                  string
+	StatusMessage               string
+	RawDuration                 int64
+	BoolAttributeKeys           []string
+	BoolAttributeValues         []bool
+	DoubleAttributeKeys         []string
+	DoubleAttributeValues       []float64
+	IntAttributeKeys            []string
+	IntAttributeValues          []int64
+	StrAttributeKeys            []string
+	StrAttributeValues          []string
+	ComplexAttributeKeys        []string
+	ComplexAttributeValues      []string
+	EventNames                  []string
+	EventTimestamps             []time.Time
+	EventBoolAttributeKeys      [][]string
```

Comment:
This group of fields should be its own struct because you will have to repeat them for Links.